File: 백준/Bronze/2420. 사파리월드/사파리월드.vb
Module Module1
    Sub Main()
        Dim N, M As Long
        Dim input() As String = Console.ReadLine().Split(" "c)
        N = Convert.ToInt64(input(0))
        M = Convert.ToInt64(input(1))

        If N < M Then
            Console.WriteLine(M - N)
        Else
            Console.WriteLine(N - M)
        End If
    End Sub
End Module

File: 백준/Bronze/2420. 사파리월드/사파리월드.swift
import Foundation

if let input = readLine() {
    let numbers = input.split(separator: " ").map { Int($0)! }
    let N = numbers[0]
    let M = numbers[1]
    
    if N < M {
        print(M - N)
    } else {
        print(N - M)
    }
}